Doxyfile

```diff
@@ -796,7 +796,10 @@ INPUT_ENCODING         = UTF-8
 # *.m, *.markdown, *.md, *.mm, *.dox, *.py, *.pyw, *.f90, *.f, *.for, *.tcl,
 # *.vhd, *.vhdl, *.ucf, *.qsf, *.as and *.js.
 
-FILE_PATTERNS          =
+FILE_PATTERNS          = *.c *.cpp *.cc *.C *.cxx *.h *.hpp *.hxx *.H \
+                         *.inc *.inl *.cu *.cuh *.m \
+                         *.idl *.f *.py *.s *.asm *.S *.ptx *.sass *.ll \
+                         *.cmake *.dox *.markdown *.md
```

Comment:
Do we really want to generate doxygen from .cmake or .md files?

Reply:
Not sure about cmake, but .md is nice to have - doxygen will format it for you.

Reply:
The thing is we already handle .md (or rather .rst that we use) through sphinx.

Reply:
.md and .markdown are handled by default anyway - i didn't want to _remove_ anything.